drivers/at86rf2xx/at86rf2xx.c

```diff
@@ -69,9 +69,7 @@ void at86rf2xx_reset(at86rf2xx_t *dev)
 
 #if CPUID_LEN < 8
     /* in case CPUID_LEN < 8, fill missing bytes with zeros */
-    for (int i = CPUID_LEN; i < 8; i++) {
-        cpuid[i] = 0;
-    }
+    memset(&(cpuid[CPUID_LEN]), 0, (IEEE802154_LONG_ADDRESS_LEN - CPUID_LEN));
```

Comment:
Is it really necessary to have all this additions in here? I think the time advantage we get from this is negligent compared to the loss of readability and some byte of ROM we loose with this method here.